src/main/java/org/polyjdbc/core/dialect/DefaultDialectTypes.java

```diff
@@ -71,4 +71,9 @@ public String bool() {
         return "BOOLEAN";
     }
 
+    @Override
+    public String binary(Object value) {
+        return "BINARY(" + value + ")";
```

Comment:
why `BINARY`? I think that most common name for binary data is CLOB or BLOB. What about other dialects?

Reply:
```
why BINARY?
```
During work, I saw a lack of support of this type. In the current database, I must save `java.util.UUID`, and I do it as `BINARY`, because it can be written in binary (`VARCHAR` isn't a good idea for this solution).

```
I think that most common name for binary data is CLOB or BLOB.
```

Right, but CLOB and BLOB are for writing for a larger data - not for a small ideas :)
However, there is no problem. If the need arises - I can add support for these types.

```
What about other dialects?
```

Oh, sorry - my fault :/ In PostgreSQL this is a [bytea](https://www.postgresql.org/docs/current/static/datatype-binary.html). Correct?

Reply:
@bartoszwalacik

Reply:
@bartoszwalacik Ok, I understand. I checked it and noticed that the mistake is only with the dialect for PostgreSQL, because there instead of `BINARY` there is `bytea`, while the rest is according to the examples in the documentation. So, I asked if I should correct it, especially since the documentation shows examples of queries like this:
```sql
SELECT E'\\000'::bytea;
```
where it is slightly different from other databases, for example - [DB2](https://www.ibm.com/support/knowledgecenter/en/SSEPEK_10.0.0/sqlref/src/tpc/db2z_bif_binary.html):
```db2
SELECT BINARY('KBH') 
     FROM SYSIBM.SYSDUMMY1;
```

Reply:
As far as I understand poly, we are talking about schema here and not about queries. We are now on GeeCON, we can meet and talk if you also here

Reply:
@bartoszwalacik Thanks for the invitation, but unfortunately I can't + I think it's a bit too late :D

Reply:
I think BINARY is ok in `DefaultDialectTypes` as it works with Oracle and MySQL. Porstgres has some crazy naming, but it can be overwritten in `PostgresDialectTypes`